apmapf.py

```diff
@@ -203,19 +283,24 @@ def run(self):
                 model = self.get_model(formula)
             except timeout_decorator.TimeoutError:
                 model = None
+            # If a model, i.e., a satisfaction of the formula, is recoverable,
+            # then we produce the solution.  This is of course within the H_MAX
+            # complexity bound.
```

Comment:
Again, fairly certain this is right, but not 100% sure.

Reply:
Yup, if the solver returns a sat result, we ask for a satisfying model. The makespan of the MAPF solution will be `H`.

Reply:
Cool, just changed `complexity` to `makespan`.  Thanks.